azkaban-web-server/src/web/js/azkaban/view/job-history.js

```diff
@@ -21,18 +21,44 @@ var jobHistoryView;
 var dataModel;
 azkaban.DataModel = Backbone.Model.extend({});
 
-$(function () {
-  var selected;
-  var series = dataSeries;
-  dataModel = new azkaban.DataModel();
-  dataModel.set({
-    "data": series
+var initJobHistoryPage = function (settings) {
+  dataModel = new azkaban.DataModel({
+    page: settings.page,
+    pageSize: settings.pageSize,
+    visiblePages: 5,
```

Comment:
why is this not part of settings?

Reply:
Added "visiblePages" to settings object to keep consistency